mesonbuild/compilers/mixins/clang.py

```diff
@@ -177,3 +177,13 @@ def get_lto_link_args(self, *, threads: int = 0, mode: str = 'default',
                 raise mesonlib.MesonException('clang support for LTO threads requires clang >=4.0')
             args.append(f'-flto-jobs={threads}')
         return args
+
+    def gen_vs_module_defs_args(self, defsfile: str) -> T.List[str]:
```

Comment:
I'm not sure to understand why this is needed, ClangCompiler inherit from GnuLikeCompiler that has this method already. You seem to just have copy/pasted code from the base class.

Reply:
That I did. Note the linked issue.

Reply:
In the issue you said "it is used for msvc and gcc, but not clang", but I don't understand why not clang, since it shared the same base class.

Reply:
The linked issue says nothing actionable. And this PR ***cannot*** be the correct solution if all it does is override an inherited method with an identical inline method.

So I agree with @xclaesse and NACK this PR unless you can explain inside meson's class hierarchy *why* this solution is technically correct.

Reply:
Could be worth writing a unit test that shows this PR actually fix something.

Reply:
This solution is not tested, true.

I have no idea why meson is not passing def files to lld.